lib/CheckToggle/index.js

```diff
@@ -48,21 +56,36 @@ class CheckToggle extends Component {
       id,
       className,
       displaySmall,
-      displayChecked
+      displayChecked,
+      spaceBetween,
```

Comment:
another utility class idea...
instead of adding this as a prop for CheckToggle would it make sense to have a `.space-between` class we can just add where ever?